driver/src/main/java/org/neo4j/driver/v1/Entry.java

```diff
@@ -21,11 +21,12 @@
 /**
  * Immutable pair of a key and a value
  *
- * @see PropertyMapAccessor
+ * @see MapAccessor
+ * @see RecordAccessor
  * @param <V> the Java type of the contained value
  */
 @Immutable
-public interface Property<V>
+public interface Entry<V>
```

Comment:
I almost always import `Entry` but I am not sure if I am representative either. `Pair`, is fine by me.